src/ride/track.c

```diff
@@ -4798,6 +4810,10 @@ void game_command_set_brakes_speed(int *eax, int *ebx, int *ecx, int *edx, int *
 	}
 
 	mapElement = map_get_first_element_at(x >> 5, y >> 5);
+	if (mapElement == NULL)
+	{
+		return;
```

Comment:
Because the game command has not been wrapped to return something, you must set ebx to the return value.
```*ebx = MONEY32_UNDEFINED```